docs/tutorial/getting-started/life-cycle/index.md

```
@@ -62,6 +62,8 @@ new Elysia()
 	.get('/2', () => 'Hello Elysia!')
 ```
 
+Here we use `status(418)` which is the "I'm a teapot" status code. You can also use the string name directly: `status("I'm a teapot")`. See <DocLink href="/tutorial/getting-started/status-and-headers">Status and Headers</DocLink> for more on using status codes.
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)